For some reason tirpc library does not provide 'key_secretkey_is_set'

--- a/src/nisplus/nis_callback.c
+++ b/src/nisplus/nis_callback.c
@@ -295,7 +295,7 @@
   cb->callback = callback;
   cb->userdata = userdata;
 
-  if ((flags & NO_AUTHINFO) || !key_secretkey_is_set ())
+  if (flags & NO_AUTHINFO)
     {
       cb->serv->key_type = NIS_PK_NONE;
       cb->serv->pkey.n_bytes = NULL;
